SpatialGDK/Extras/internal-documentation/working-sets/edge-case-1.wsd

```diff
@@ -0,0 +1,47 @@
+@startuml Working sets - all on the same ServerWorker
```

Comment:
These are great, but could you include a short explaining file detailing how to view them correctly in `internal-documentation` ?

Reply:
Definitely - I'll add a README file.